Source/SwiftLintFramework/Rules/RuleConfigurations/FileNameConfiguration.swift

```diff
@@ -6,10 +6,15 @@ public struct FileNameConfiguration: RuleConfiguration, Equatable {
 
     private(set) public var severity: SeverityConfiguration
     private(set) public var excluded: Set<String>
+    private(set) public var prefixPattern: String
```

Comment:
I'm wondering if it makes sense to make those `[String]` so you could easily support multiple patterns without having to combine regexes manually. Probably overengineering though.

Reply:
I thought about that too, but I thought exactly the same. I opted to wait until there's somebody who has an actual use case to prevent overengineering.

Reply:
👍